### ISSUE TITLE
yarn test command fails

### ISSUE THREAD

**Issue by iokhotnikov** [reimagined/resolve#675]
**Describe the bug**
`yarn test` command fails

**To Reproduce**
```zsh
 ✘ ilyaokhotnikov  ~/Projects/resolve   master  yarn test
yarn run v1.7.0
$ oao all "yarn test --color 2>&1"
2018-07-03T07:45:43.715Z           storyboard INFO  ┌── ROOT STORY: Node.js 9.3.0 on Darwin 64-bit, SB 3.1.3 [CREATED]
2018-07-03T07:45:43.716Z           storyboard INFO  Log filter: *:DEBUG
2018-07-03T07:45:43.893Z                 main INFO  Reading all package.json files...
2018-07-03T07:45:43.896Z                 main INFO  ┌── Run cmd yarn test --color 2>&1 at packages/create-resolve-app [CREATED]
$ exit 0 --color
2018-07-03T07:45:44.275Z                 yarn INFO  /bin/sh: line 0: exit: too many arguments
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
2018-07-03T07:45:44.288Z                 yarn INFO
2018-07-03T07:45:44.290Z                 main ERROR Command 'yarn test --color 2>&1' failed [1] at packages/create-resolve-app
2018-07-03T07:45:44.290Z                 main INFO  └── Run cmd yarn test --color 2>&1 at packages/create-resolve-app [CLOSED]
{ Error: Command 'yarn test --color 2>&1' failed [1] at packages/create-resolve-app
    at execError (/Users/ilyaokhotnikov/Projects/resolve/node_modules/oao/lib/utils/shell.js:193:13)
    at _callee2$ (/Users/ilyaokhotnikov/Projects/resolve/node_modules/oao/lib/utils/shell.js:173:20)
    at tryCatch (/Users/ilyaokhotnikov/Projects/resolve/node_modules/babel-polyfill/node_modules/regenerator-runtime/runtime.js:65:40)
    at Generator.invoke [as _invoke] (/Users/ilyaokhotnikov/Projects/resolve/node_modules/babel-polyfill/node_modules/regenerator-runtime/runtime.js:303:22)
    at Generator.prototype.(anonymous function) [as throw] (/Users/ilyaokhotnikov/Projects/resolve/node_modules/babel-polyfill/node_modules/regenerator-runtime/runtime.js:117:21)
    at step (/Users/ilyaokhotnikov/Projects/resolve/node_modules/oao/lib/utils/shell.js:28:191)
    at /Users/ilyaokhotnikov/Projects/resolve/node_modules/oao/lib/utils/shell.js:28:402
    at <anonymous>
    at process._tickCallback (internal/process/next_tick.js:160:7)
  code: 1,
  stdout: '\u001b[2K\u001b[1G\u001b[2m$ exit 0 --color\u001b[22m\n/bin/sh: line 0: exit: too many arguments\n\u001b[2K\u001b[1G\u001b[31merror\u001b[39m Command failed with exit code 1.\n\u001b[2K\u001b[1G\u001b[34minfo\u001b[39m Visit \u001b[1mhttps://yarnpkg.com/en/docs/cli/run\u001b[22m for documentation about this command.\n',
  stderr: '' }
2018-07-03T07:45:44.292Z           storyboard INFO  └── ROOT STORY: Node.js 9.3.0 on Darwin 64-bit, SB 3.1.3 [CLOSED]
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.

```

 - MacOS 10.12.6
 - Node.js 9.3.0
 - yarn 1.7.0

**Comment by TatyanaRyzh** [comment]
Hi iokhotnikov,
 
We've reproduced your scenario and will fix it as soon as possible.
 
Thank you for helping us improve reSolve!

**Comment by TatyanaRyzh** [comment]
Fixed in [0.13.0](https://github.com/reimagined/resolve/releases/tag/V0.13.0)
